### DISCUSSION TITLE
Jake Cooper: Railway

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#77] [Announcements]
## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: Hey everyone! Our guest today is Jake Cooper, the founder of Railway.app. Before we talk about Railway, Jake would you like to tell our listeners a bit more about yourself?

## Questions

- [x] What is railway?
- [x] Where did the idea for railway start? What personal need were you trying to solve/itch?
- [x] How does railway differ services like heroku or render?
- [x] What is "Infrastructure as Legos"? How does this simplify building applications?
- [x] How does railway make it easier to iterate and ship code?
- [x] How does railway handle auto-scaling? Do I have to write my code differently?
- [x] Does railway support monorepos?
- [x] What feature of railway are you most proud of? Something that makes you and your users have that "aha" moment?
- [x] What's the hardest technical problem you've had to solve at Railway?
- [x] What are you building towards? Are there any cool features users can look forward to?
- [x] As a first time founder how hard was it getting railway off the ground? Do you have any insights into getting VC capital for dev tooling startup? Why did you choose to go the VC route instead of bootstrapping?
- [ ] What parts of software development (or the world) do you think are ripe for innovation through automation?
- [ ] What technology has you most excited for the future of development?

## Outro

Andrew: Well that’s it for this week’s episode of devtools.fm. Be sure to follow us on YouTube and wherever you consume your podcasts. Thanks for listening!

**Comment by hipstersmoothie** [comment]
## Research Links

- https://twitter.com/JustJake
- https://twitter.com/Railway
- https://railway.app
- https://justjake.substack.com

## Previous Work

- Uber
- https://twitter.com/pioneerdotapp
- https://twitter.com/WolframResearch

**Comment by just-be-dev** [comment]
My tooltips:

- https://github.com/spacedriveapp/spacedrive
- https://github.com/actualbudget/actual
- https://watchy.sqfmi.com/

**Comment by hipstersmoothie** [comment]
Tooltip:

https://docs.plasmo.com
https://lion-web.netlify.app

**Comment by JakeCooper** [comment]
https://tamagui.dev/
https://tailscale.com/
https://github.com/graphql-nexus/nexus && https://trpc.io/
